2017-00-00-shipping-rust-1.md

```diff
@@ -0,0 +1,70 @@
+# Shipping Rust on Tessel
+
+***Tessel is an open-source hardware platform for developing IoT devices.***
+
+The Tessel board runs an embedded Linux distribution (OpenWRT) and uses a less-common chip architecture: MIPS with soft-float.
```

Comment:
Who is the intended audience for this post? This is a tough intro for a general audience, but doesn't mention Rust right away so might not reach that reader either